frameworks/Java/wizzardo-http/wizzardo-http-base.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM techempower/gradle-java8:0.1
```

Comment:
Please use `FROM gradle:4.6.0-jdk8` instead, assuming that works.

Very recently we rewrote all our dockerfiles to avoid `techempower/*` dependencies and to favor "official" or widely-used dependencies from docker hub instead.  We realized that we don't want the responsibility of publishing and maintaining our own set of artifacts if we can avoid it.

Here's another Java framework that depends on gradle as an example:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/d018a0df30e6e3ed92d390915dd61c5fabd4f1b9/frameworks/Java/jawn/jawn.dockerfile